storage/backup/backup-cli/src/metadata/cache.rs

```diff
@@ -138,27 +161,22 @@ pub async fn sync_and_load(
     NUM_META_DOWNLOAD.set(0);
     let futs = new_remote_hashes.iter().enumerate().map(|(i, h)| {
         let fh_by_h_ref = &remote_file_handle_by_hash;
-        let storage_ref = &storage;
+        let storage_ref = storage.as_ref();
         let cache_dir_ref = &cache_dir;
 
         async move {
             let file_handle = fh_by_h_ref.get(*h).expect("In map.");
             let local_file = cache_dir_ref.join(*h);
             let local_tmp_file = cache_dir_ref.join(format!(".{}", *h));
-            // download to tmp file ".xxxxxx"
-            tokio::io::copy(
-                &mut storage_ref
-                    .open_for_read(file_handle)
-                    .await
-                    .err_notes(file_handle)?,
-                &mut OpenOptions::new()
-                    .write(true)
-                    .create_new(true)
-                    .open(&local_tmp_file)
-                    .await
-                    .err_notes(&local_file)?,
-            )
-            .await?;
+
+            // Read the metadata file. If the metadata file not exist, there is a chance it is moved to backup folder while compaction is in progress
```

Comment:
We don't need this fallback logic any more, right?

Reply:
This is assuming the user set a very short expiration time by mistake, this would work as a fallback fault recovery in this case.

Reply:
removed